packages/web-ui/src/Box/Box.docs.mdx

```diff
@@ -0,0 +1,59 @@
+import {ArgsTable, Meta, Canvas } from '@storybook/blocks';
+import * as BoxStories from './Box.stories';
+import Box from './Box.tsx'
+import {Text} from '../Typography'
+import ThemeProvider from '../ThemeProvider'
+
+<Meta of={BoxStories} />
+
+# Box
+
+Box is a low-level primitive, which supports theme-aware styling props, and can
+be used for building any styled element.
+
+* [Semantic HTML](#semantic-html)
+* [Background](#background)
+* [Props](#props)
+
+## Semantic HTML
+
+By default Box renders a div element, this can be customised using the
+`component` prop.
+
+```tsx
+<Box component="span">
+  ...
+</Box>
+```
+
+## Background
+
+Additionally to the system properties, Box supports pre-defined backgrounds
+that provide visual context to child components. This means that components
+will change colours depending on the parent background.
```

Comment:
maybe here we can specify and say `will change foreground colours` just to make it more designer friendly